website/docs/api/regions.mdx

```diff
@@ -60,7 +60,7 @@ function example() {
 <TabItem value="python">
 
 ```py
-h3.polygon_to_cells(polygons, res, geo_json_conformant=False)
+h3.h3shape_to_cells(h3shape, res)
```

Comment:
`polygon_to_cells` does exist in the new version of `h3-py`, though. Wouldn't that make more sense to put here?

Reply:
I had wanted to direct folks to `h3shape_to_cells`, since `polygon_to_cells` is a weird duck in Python, but maybe that's what the `h3-py` docs are for :)

I can have it be `polygon_to_cells` here.

Reply:
I think your compromise solution of having both API calls mentioned is fine here.